From 63af44ba75c721b8bbfca88090fbe7a04ffb4182 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Mon, 3 Aug 2015 18:02:29 +0800
Subject: [PATCH] MLK-11310 ARM: dts: update cpufreq setpoint for imx6sx

According to latest datasheet, Rev.1, 7/2015, the LDO
bypass voltage for 996MHz setpoint on some of our boards
are NOT matching this datasheet, update them accordingly.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/boot/dts/imx6sx-17x17-arm2.dts | 14 ++++++++++++++
 arch/arm/boot/dts/imx6sx-19x19-arm2.dts | 14 ++++++++++++++
 arch/arm/boot/dts/imx6sx-sdb-reva.dts   |  2 +-
 3 files changed, 29 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
index 948c93a..92a29e1 100644
--- a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
@@ -121,6 +121,20 @@
 };
 
 &cpu0 {
+	operating-points = <
+		/* kHz    uV */
+		996000  1275000
+		792000  1175000
+		396000  1075000
+		198000	975000
+		>;
+	fsl,soc-operating-points = <
+		/* ARM kHz      SOC uV */
+		996000	1175000
+		792000	1175000
+		396000	1175000
+		198000	1175000
+	>;
 	arm-supply = <&sw1a_reg>;
 	soc-supply = <&sw1c_reg>;
 };
diff --git a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
index 99478eb..7f63da4 100644
--- a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
@@ -110,6 +110,20 @@
 };
 
 &cpu0 {
+	operating-points = <
+		/* kHz    uV */
+		996000  1275000
+		792000  1175000
+		396000  1075000
+		198000	975000
+		>;
+	fsl,soc-operating-points = <
+		/* ARM kHz      SOC uV */
+		996000	1175000
+		792000	1175000
+		396000	1175000
+		198000	1175000
+	>;
 	arm-supply = <&sw1a_reg>;
 	soc-supply = <&sw1c_reg>;
 };
diff --git a/arch/arm/boot/dts/imx6sx-sdb-reva.dts b/arch/arm/boot/dts/imx6sx-sdb-reva.dts
index e8fa36b..3805fc5 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-reva.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-reva.dts
@@ -17,7 +17,7 @@
 &cpu0 {
 	operating-points = <
 		/* kHz    uV */
-		996000  1250000
+		996000  1275000
 		792000  1175000
 		396000  1075000
 		198000	975000
-- 
1.8.0

